python/ml4ir/base/features/feature_fns/sequence.py

```diff
@@ -0,0 +1,41 @@
+import tensorflow as tf
+from tensorflow.keras import layers
+from tensorflow import io
+
+
+def bytes_sequence_to_encoding(feature_tensor, feature_info):
+    """Encode a sequence of numbers into a fixed size tensor"""
+    feature_layer_info = feature_info["feature_layer_info"]
+
+    # Decode string tensor to bytes
+    feature_tensor = io.decode_raw(
+        feature_tensor, out_type=tf.uint8, fixed_length=feature_layer_info["args"]["max_length"],
+    )
+
+    feature_tensor = tf.reshape(feature_tensor, [-1, feature_layer_info["args"]["max_length"]])
+    if "embedding_size" in feature_layer_info["args"]:
+        char_embedding = layers.Embedding(
+            input_dim=256,
```

Comment:
Do we plan to have input_dim parametrizable? Should we make it global variable as it appears in both conditions?